src/components/take-test/TakeTest.tsx

```diff
@@ -5,37 +5,17 @@ import Test from './Test'
 import Paper from '../Paper'
 
 export default ({ test, setRedirect, id }: any) => {
-  const { updateAirDB, postAirDB, users, scores } = React.useContext(
-    AirDBContext
-  )
+  const { handleSetScore, createInitialScore } = React.useContext(AirDBContext)
```

Comment:
Good. As far as naming goes, try to come up with something that doesn't just say "handleThis" and "handleThat." The name `createInitialScore` is excellent.